src/use-cases/commands/AccepterUnMembre.ts

```diff
@@ -0,0 +1,64 @@
+import { CommandHandler, ResultAsync } from '../CommandHandler'
+import { GetGouvernanceRepository } from './shared/GouvernanceRepository'
+import { GetMembreRepository, UpdateMembreRepository } from './shared/MembreRepository'
+import { GetUtilisateurRepository } from './shared/UtilisateurRepository'
+import { GouvernanceUid } from '@/domain/Gouvernance'
+import { Membre, MembreFailure } from '@/domain/Membre'
+
+export class AccepterUnMembre implements CommandHandler<Command> {
+  readonly #utilisateurRepository: UtilisateurRepository
+  readonly #gouvernanceRepository: GouvernanceRepository
+  readonly #membreRepository: MembreRepository
+
+  constructor(
+    utilisateurRepository: UtilisateurRepository,
+    gouvernanceRepository: GouvernanceRepository,
+    membreRepository: MembreRepository
+  ) {
+    this.#utilisateurRepository = utilisateurRepository
+    this.#gouvernanceRepository = gouvernanceRepository
+    this.#membreRepository = membreRepository
+  }
+
+  async handle(command: Command): ResultAsync<Failure> {
+    const gestionnaire = await this.#utilisateurRepository.get(command.uidGestionnaire)
+
+    const membreInvalide = await this.#membreRepository.get(command.uidMembreInvalide)
+
+    const gouvernance = await this.#gouvernanceRepository.get(new GouvernanceUid(command.uidGouvernance))
+
+    if (!membreInvalide.appartientALaGouvernance(command.uidGouvernance)) {
+      return 'membreInvalideNonAssocieALaGouvernance'
+    }
+
+    if (!gouvernance.peutEtreGerePar(gestionnaire)) {
```

Comment:
La méthode devrait être renommée :
```suggestion
    if (!gouvernance.peutEtreGereePar(gestionnaire)) {
```